docker-compose.dev.yml

```diff
@@ -115,5 +116,11 @@ services:
     restart: on-failure
     networks:
       - internal
+    healthcheck:
+      # "clickhouse", "client", "-u ${CLICKHOUSE_USER}", "--password ${CLICKHOUSE_PASSWORD}", "-q 'SELECT 1'"
+      test: wget --no-verbose --tries=1 http://127.0.0.1:8123/ping || exit 1
+      interval: 10s
```

Comment:
Do we want the interval to be that high? I imagine if they're all started around the same time, we could get away with a shorter timeframe.

Reply:
Yeah, I just picked a random value. Playing with it on my laptop 3 seconds also fixes the race.